airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_source.py

```diff
@@ -106,26 +106,28 @@ def test_spec(self):
 
     def test_get_custom_insights_streams(self, api, config):
         config["custom_insights"] = [
-            {"name": "test", "fields": ["account_id"], "breakdowns": ["ad_format_asset"], "action_breakdowns": ["action_device"]},
+            {"name": "test", "fields": ["account_id"], "breakdowns": ["ad_format_asset"], "action_breakdowns": ["action_device"], "action_report_time": "mixed"},
```

Comment:
After making the changes, does backward compatibility still exist?

Reply:
> What expected if the configuration does not have an "action_report_time" parameter?

if an `action_report_time` parameter isn't present, `mixed` would be the value because this is the default value for this field

> After making the changes, does backward compatibility still exist?

I think so because it's the default value that is set in tests.

Reply:
However, if the user already has a configuration without those parameters, and it is not possible to set a default value.

Reply:
However, if the user already has a configuration without those parameters, and it is not possible to set a default value.

Reply:
okay! done :+1: